Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Component to simply play a descending animation when this NetworkObject's parent NetworkObject changes.
+    /// </summary>
+    public class ClientPickUpPotVisualizer : NetworkBehaviour
```

Comment:
It's not just "visualizer" if there's sound as well :/

Reply:
would something like ClientPickUpPotEffects work better?

Reply:
tho I'll also raise that the ClientTossedItemVisualizer script also has an AudioSource reference too 😅

Reply:
🚀

Reply:
> tho I'll also raise that the ClientTossedItemVisualizer script also has an AudioSource reference too 😅

Might be worth updating it as well @fernando-cortez

Reply:
Added a task for this! [MTT-4654](https://jira.unity3d.com/browse/MTT-4654).